pkg/sql/schemachanger/rel/schema_rules.go

```diff
@@ -87,19 +100,29 @@ var (
 	clausesType = reflect.TypeOf((*Clauses)(nil)).Elem()
 )
 
+func makeRandomVars(n int) (ret []Var) {
+	ret = make([]Var, n)
+	for i := range ret {
+		ret[i] = Var(hex.EncodeToString(uuid.MakeV4().GetBytes()))
```

Comment:
This is going to hurt legibility when debugging. Can't we instead use a reserved prefix followed by a counter? You probably can assert that this reserved prefix isn't used in vars outside of rel.